docs/design-docs/sync.md

```diff
@@ -0,0 +1,60 @@
+# Sync WordPress Studio Sites
+
+## About this doc
+
+This design document describes the requirements, architecture, and data models for the **Studio Sync** feature.
+
+## Context
+
+WordPress Studio Sync enables developers to pull a live site down for local development (including themes, plugins, media, and the database) and then push changes back without manual exports or FTP. It supports both WordPress.com and Pressable sites with a Jetpack connection.
+
+## Terminology
+
+- **Sync**: Replicating files and database content between a local machine and a remote site in any direction.
+- **Push**: Copying changes from a local machine to a remote (staging or production) site.
+- **Pull**: Copying changes from a remote site down to a local machine.
+- **Staging site**: A staging site is hosted on WordPress.com and is connected to its production site. It is used to test the sync feature and serves as the source of truth for this feature. WordPress.com staging sites can sync with production sites and vice versa.
+- **Production site**: A production site is hosted on WordPress.com and is used to store the synced site. We consider all Pressable sites as production sites.
```

Comment:
I think it's worth clarifying why we consider all Pressable sites production sites. Pressable does offer staging sites, so the question can easily come up.

Reply:
I've updated the sentence bd788a5abdc1058eb781ef05c86f5db565571750. We may be able to distinguish between staging and production for Pressable, so we'll update these definitions later in the project if necessary. Thanks for the suggestion.